README.md

```diff
@@ -44,7 +44,7 @@ VolumeReplicationClass corresponding to the driver providing replication.
 
 `volumeReplicationClass` is the class providing replication
 
-`imageState` is the state of the image being referenced. Possible values are `primary`. `secondary` and `resync`.
+`replicationState` is the state of the image being referenced. Possible values are `primary`. `secondary` and `resync`.
```

Comment:
```suggestion
`replicationState` is the state of the volume being referenced. Possible values are `primary`. `secondary` and `resync`.
```
If we're not mentioning "image", should we also make this change at all places?

Reply:
yep. done.